machine-learning/lambda-v2/lambda-artifacts/com.lambda.DLRImageClassification/1.0.0/image_classification/prediction_utils.py

```diff
@@ -0,0 +1,162 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import platform
+from ast import literal_eval
+from datetime import datetime
+from io import BytesIO
+from json import dumps
+from sys import modules
+from time import sleep
+
+import config_utils
+import IPCUtils as ipc_utils
+from cv2 import VideoCapture, destroyAllWindows, imdecode, imread, resize
+from dlr import DLRModel
+from numpy import argsort, frombuffer, fromstring, load, uint8
+
+config_utils.logger.info("Using dlr from '{}'.".format(modules[DLRModel.__module__].__file__))
+config_utils.logger.info("Using np from '{}'.".format(modules[argsort.__module__].__file__))
+config_utils.logger.info("Using cv2 from '{}'.".format(modules[VideoCapture.__module__].__file__))
+
+# Read synset file
+with open(config_utils.LABELS, "r") as f:
+    synset = literal_eval(f.read())
+
+dlr_model = DLRModel(config_utils.MODEL_DIR, config_utils.DEFAULT_ACCELERATOR)
+
+
+def enable_camera():
+    if platform.machine() == "armv7l":  # RaspBerry Pi
+        import picamera
+
+        config_utils.CAMERA = picamera.PiCamera()
+    elif platform.machine() == "aarch64":  # Nvidia Jetson TX
+        config_utils.CAMERA = VideoCapture(
+            "nvarguscamerasrc ! video/x-raw(memory:NVMM),"
+            + "width=(int)1920, height=(int)1080, format=(string)NV12,"
+            + "framerate=(fraction)30/1 ! nvvidconv flip-method=2 !"
+            + "video/x-raw, width=(int)1920, height=(int)1080,"
+            + "format=(string)BGRx ! videoconvert ! appsink"
+        )
+    elif platform.machine() == "x86_64":  # Deeplens
+        import awscam
+
+        config_utils.CAMERA = awscam
+
+
+def predict(image_data):
+    r"""
+    Performs image classification and predicts using the model.
+
+    :param image_data: numpy array of the resized image passed in for inference.
+    """
+    PAYLOAD = {}
+    PAYLOAD["timestamp"] = str(datetime.now())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)